vrx_gazebo/worlds/xacros/gymkhana.xacro

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
+<world xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="gymkhana" params="name='gymkhana' 
+   nav_uri:=short_navigation_course0
+   obstacle_uri:=obstacle_course
+   nx:=-524 ny:=186 nz:=0 nR:=0 nP:=0 nY:=-1.44 
+   ox:=-477 oy:=275 oz:=0 oR:=0 oP:=0 oY:=2.54 
+   gx:=-483 gy:=295.5 
+   running_duration:=300 **gates">
```

Comment:
Is this parameter used?

Reply:
Which one? running_duration or gates?

Reply:
running_duration is not currently being used but could be useful to change during testing. Likewise, gates has the same value for everything right now because that is what matches the practice worlds we designed so far, but I could imagine varying the values in the future.

Reply:
Ah! I see what you mean. I forgot to put the variable in. Will fix that now.

Reply:
It's OK to not use it and fallback on the default value but what I mean is that `running_duration` should appear in all the plugins as `<running_state_duration>${running_duration}</running_state_duration>`, right?

Reply:
Yes! I think our messages are missing each other slightly but I understand and just fixed this.